experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -906,6 +910,66 @@ function root_system(w::WeightLatticeElem)
   return w.root_system
 end
 
+###############################################################################
+# more functions
+
+function dot(r::RootSpaceElem, w::WeightLatticeElem)
+  @req root_system(r) === root_system(w) "parent root system mismatch"
+
+  return sum(
+    r[i] * cartan_symmetrizer(cartan_matrix(root_system(r)); check=false)[i] * w[i] for
```

Comment:
We wanted to cache the symmetrizer, right? Either way it shouldn't be in the sum expression.

Reply:
Oops, yeah. Forgot that :sweat_smile: